### PR TITLE
edit readme for fccLndData

### PR DESCRIPTION
**Pull request by dlewis87**

### CODE REVIEW

File: dev/app/data/README.md [hunk 1]
```diff
@@ -25,38 +27,38 @@ Here some tips on how fill fccLndDat.json:
...
+            "pic":"http://external-image-host.com/your-image",
             "email":"something@gmail.com",
-            "fcc":"http://google.com",
-            "tw":"http://google.com",
-            "lk":"http://google.com",
-            "web":"http://google.com",
-            "fb":"http://google.com",
+            "fcc":"http://freecodecamp.com/camper",
+            "tw":"http://twitter.com/camper",
+            "lk":"http://linedin.com/camper",
```
Comment (chris-alexander) [inline dev/app/data/README.md]: I think this should be `https://uk.linkedin.com/in/camper`

File: dev/app/data/README.md [hunk 2]
```diff
@@ -89,11 +91,11 @@ Just remember if you are inserting a link to add:
 
 **name** - your name -> REQUIRED
 
-**class** - fill in with class corrisponding FCC sessions you have completed, that's use for filter your profile on camper page -> REQUIRED for references see at [Classes you can use](#classes-you-can-use)
+**class** - list of all classes corrisponding to the FCC sessions you have completed, these are used for the filter on the camper page so the user can select all campers at a particular point -> REQUIRED for references see at [Classes you can use](#classes-you-can-use)
 
 **bio**- your presentation -> REQUIRED - max 120 words !!!!!
```
Comment (chris-alexander) [inline dev/app/data/README.md]: would a character limit be better and more easily tested?

File: dev/app/data/README.md [hunk 3]
```diff
@@ -117,17 +119,17 @@ Just remember if you are inserting a link to add:
 
 ####*Classes you can use:*####
 
-The first level of filtering highlight which of the major FCC session Campers is working or has been working at. Every campers should have one or more. Please insert the session you are working on along with the past one if you are available to help other people with those.
+The first level of filtering highlights which of the main FCC session campers are on or have been working on. Every camper should have at least one of the main classes. Please add the exercise/project you are working on, along with any previous ones that you are available to help other people with.
 
 #####*FILETERING FIRST LEVEL:*#####
```
Comment (chris-alexander) [inline dev/app/data/README.md]: Spelling: Filtering

File: dev/app/data/README.md (hunk 2 above)
Comment (artitudinale1) [inline dev/app/data/README.md]: 250 characters ?

**Comment by artitudinale1** [comment]
I can't checkout fix/issue#18 - did the branch has been actually push up?
I think at least us should push branch up - so other ppl can edit and merge it especially if just spelling mistake or obvious stuff what you think?

File: dev/app/data/README.md (hunk 2 above)
Comment (dlewis87) [inline dev/app/data/README.md]: Yeah should be enough, only a rough guide anyway...

**Comment by artitudinale1** [comment]
I don't mind amend this and push up changes but I can't checkout fix/issue#18. Probably it is in @silverwedding fork?

**Comment by dlewis87** [comment]
Yeh it's in my fork, I've made the amendments now. 👍

**Comment by artitudinale1** [comment]
good to go then ! We can even change/add later !

**Comment by artitudinale1** [comment]
I added link to it from main readme . it d be more visible

**Pull request merged**